XRTK-Core/Packages/com.xrtk.core/Runtime/Providers/Controllers/Simulation/Hands/SimulatedHandDataConverter.cs

```diff
@@ -0,0 +1,292 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using XRTK.Definitions.Controllers.Hands;
+using XRTK.Definitions.Controllers.Simulation.Hands;
+using XRTK.Definitions.Utilities;
+using XRTK.Providers.Controllers.Hands;
+using XRTK.Utilities;
+using XRTK.Services;
+using Random = UnityEngine.Random;
+
+namespace XRTK.Providers.Controllers.Simulation.Hands
+{
+    /// <summary>
+    /// Hand controller type for simulated hand controllers.
+    /// </summary>
+    public sealed class SimulatedHandDataConverter
```

Comment:
What's the purpose of making it `sealed`?

Reply:
Because I don't want people to inherit this class, it's not supposed to.